Library.ino

```diff
@@ -42,19 +42,22 @@ bool quit; // Used for runsketch()
     bool inRegion(int Y, int Bot, int Top = 0, int X = 120, int Left = 0, int Right = 240); //If given coordinates are in the given range, returns true.
     int calculateListY(int num); // Calculates the Y position of a button with the given number that is in a list (not used by the library), you can use it.
     void start(); // Starts TFT screen and draws page 0. Call only once and at the end of setup().
-    void navigatePage( int page , int transition = 1); // Navigates to a page. This can be used in loop(). It is implicitly called by start() and recalling it causes another draw (only waste of time)
+    void navigatePage( int page , int transition = DEFAULT); // Navigates to a page. This can be used in loop(). It is implicitly called by start() and recalling it causes another draw (only waste of time)
     #ifdef BUTTON // Button functions
         void drawButton(int page,int i); // Draws a button. Don't use publicly. It will draw even if the button is not in current page.
-        int addbutton(int page, uint16_t X, uint16_t Y, uint16_t Width, uint16_t Height, String text,
-        uint16_t textcolor=TFT_WHITE, uint16_t backcolor=TFT_DARKGREY, uint16_t bordercolor=AUTO, bool enabled = true, 
-        bool visible = true,int radius=0,int XtextOffset=0,int YtextOffset=0) ; // Adds a button. Use before start()
+        int addbutton( int page, uint16_t X, uint16_t Y, uint16_t Width, uint16_t Height, String text,
+                   uint16_t textcolor = TFT_WHITE, uint16_t backcolor = TFT_DARKGREY, uint16_t bordercolor = AUTO, bool enabled = true, 
+                   bool visible = true,int radius=0,int XtextOffset=0,int YtextOffset=0 )  ; // Adds a button. Use before start()
         void undrawButton(int page,int i); // Hides a button. Don't use publicly
         void changeButtonProperty(int page, int i, int propertyName, int val,int a=0); // Changes a button's property
         void changeButtonProperty(int page, int i, int propertyName, bool val,bool a=false); // Changes a button's property
         void changeButtonProperty(int page, int i, int propertyName, String val, String a = ""); // Changes a button's propery
     #endif
     #ifdef LABEL  // Label functions
-        int  addlabel(int page, uint16_t X, uint16_t Y, String text, uint16_t textcolor, const GFXfont *font = NULL, byte textsizeX = 1, byte textsizeY = 1, bool enabled = true, bool visible = true); // Adds a label. Use before start()
+        int addlabel(int page, uint16_t X, uint16_t Y, String text,
+                     uint16_t textcolor, const GFXfont *font = NULL,
+                     byte textsizeX = 1, byte textsizeY = 1,
+                     bool enabled = true, bool visible = true); // Adds a label. Use before start()

```

Comment:
Here, when I want to remove the default argument for `visible`, the compiler says `GFXfont does not name a type` but will compile with the default argument.